specification/search/resource-manager/Microsoft.Search/stable/2023-11-01/examples/GetQuotaUsagesList.json

```diff
@@ -0,0 +1,86 @@
+                            "localizedValue": "F - Free"
+                        }
+                    },
+                    {
+                        "id": "/subscriptions/{subscriptionId}/providers/Microsoft.Search/locations/{location}/usages/basic",
+                        "unit": "Count",
+                        "currentValue": 8,
+                        "limit": 16,
+                        "name": {
+                            "value": "basic",
+                            "localizedValue": "B - Basic"
+                        }
+                    },
+                    {
+                        "id": "/subscriptions/{subscriptionId}/providers/Microsoft.Search/locations/{location}/usages/standard",
+                        "unit": "Count",
+                        "currentValue": 8,
+                        "limit": 16,
+                        "name": {
+                            "value": "standard",
+                            "localizedValue": "S - Standard"
+                        }
+                    },
+                    {
+                        "id": "/subscriptions/{subscriptionId}/providers/Microsoft.Search/locations/{location}/usages/standard2",
+                        "unit": "Count",
+                        "currentValue": 8,
+                        "limit": 16,
+                        "name": {
+                            "value": "standard2",
+                            "localizedValue": "S2 - Standard2"
+                        }
+                    },
+                    {
+                        "id": "/subscriptions/{subscriptionId}/providers/Microsoft.Search/locations/{location}/usages/standard3",
+                        "unit": "Count",
+                        "currentValue": 8,
+                        "limit": 16,
+                        "name": {
+                            "value": "standard3",
+                            "localizedValue": "S3 - Standard3"
+                        }
+                    },
+                    {
+                        "id": "/subscriptions/{subscriptionId}/providers/Microsoft.Search/locations/{location}/usages/storageOptimizedL1",
+                        "unit": "Count",
+                        "currentValue": 8,
+                        "limit": 16,
+                        "name": {
+                            "value": "storageOptimizedL1",
+                            "localizedValue": "L1 - Storage Optimized"
+                        }
+                    },
+                    {
+                        "id": "/subscriptions/{subscriptionId}/providers/Microsoft.Search/locations/{location}/usages/storageOptimizedL2",
+                        "unit": "Count",
+                        "currentValue": 8,
+                        "limit": 16,
+                        "name": {
+                            "value": "storageOptimizedL2",
+                            "localizedValue": "L2 - Storage Optimized"
```

Comment:
How were these localized values determined?

Reply:
The ARM spec for Usages api expects to specify a "LocalizableString object", but we only have these string definitions for the skus we support hence went ahead with the same.